config/config.exs

```diff
@@ -28,3 +28,5 @@ use Mix.Config
 # here (which is why it is important to import them last).
 
 import_config "#{Mix.env()}.exs"
+
+config :tesla, adapter: Tesla.Adapter.Hackney
```

Comment:
Hereby I summon @britto, I think he had a couple of things to say about hackney 😄
Maybe we could use [`mint`](https://github.com/elixir-mint/mint)?